dashboard-ui-v2/src/components/config/mount-pod-patch-form.tsx

```diff
@@ -200,6 +203,130 @@ const MountPodPatchForm: React.FC<{
                 </ProForm.Item>
               ),
             },
+            {
+              title: 'hostNetwork',
+              key: 'hostNetwork',
+              render: () => (
+                <ProForm.Item
+                  className="patch-form-checkbox-item"
+                  name={'hostNetwork'}
+                >
+                  <ProFormCheckbox name={'hostNetwork'} />
+                </ProForm.Item>
+              ),
+            },
+            {
+              title: 'hostPID',
+              key: 'hostPID',
+              render: () => (
+                <ProForm.Item
+                  className="patch-form-checkbox-item"
+                  name={'hostPID'}
+                >
+                  <ProFormCheckbox name={'hostPID'} />
+                </ProForm.Item>
```

Comment:
how to unset for use the default value

Reply:
remove it in yaml?

Reply:
Keep the current implementation in form and show `-` in display